core/mem/allocators.odin

```diff
@@ -749,6 +749,7 @@ dynamic_pool_alloc_bytes :: proc(p: ^Dynamic_Pool, bytes: int) -> ([]byte, Alloc
 	n := bytes
 	extra := p.alignment - (n % p.alignment)
 	n += extra
+    if n > p.block_size do return nil, .Invalid_Argument
```

Comment:
Please use tabs for indentation.

Reply:
Whoops, that was quite literally invisible on my side. Fixed now.